docs/products/redis/concepts/warning-overcommit_memory.rst

```diff
@@ -0,0 +1,8 @@
+Warning ``overcommit_memory``
+=============================
+
+By starting a Redis service on `Aiven console <https://console.aiven.io/>`_, you may notice on **Logs** the following **warning** ``overcommit_memory``::
+
+    # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
+
+This warning can be safely ignored. Aiven for Redis limits the memory usage differently from how the software is deployed, which means the available memory will never drop low enough to hit this particular failure case.
```

Comment:
"from how the software is deployed," - what do we mean by this? Is it different because we manage it? I'm not a Redis person, but I think is a bit wage explanation to understand.

Reply:
It is meant to be like: from the default use of the software without aiven. I also do not think needs to be added, so I can drop it out.